hnn_core/network.py

```diff
@@ -216,47 +245,85 @@ def __repr__(self):
                  len(self.pos_dict['L5_basket'])))
         return '<%s | %s>' % (class_name, s)
 
-    def _add_external_feeds(self):
-        """Legacy function, for backward compatibility with original HNN-GUI
+    def attach_drive(self, drive_name, drive_params=None,
+                     cell_specific=False):
```

Comment:
shouldn't `feed_times` be an argument to this method? And also the connectivity parameters? They can be stored in `Network` and instantiated in `NetworkBuilder`

Reply:
My thought goes like this: 'attaching' drives (same for biases) is defining the parameters (event time _statistics_ and connectivity within the network). 'instantiating' the drives then amounts to filling the `drive_event_times` dictionary with actual event times. Correspondingly, you can 'attach' a bias (basically just check the params are sane), but wait until `cell.create_tonic_bias` in `NetworkBuilder` before actually creating any HOC objects.